comp/core/secrets/secretsimpl/secrets.go

```diff
@@ -144,9 +151,15 @@ func (r *secretResolver) fillFlare(fb flaretypes.FlareBuilder) error {
 	r.GetDebugInfo(writer)
 	writer.Flush()
 	fb.AddFile("secrets.log", buffer.Bytes())
+
+	content, err := filesystem.ReadFileWithSizeLimit(r.auditFilename, int64(r.auditFileMaxSize))
+	if err == nil {
+		fb.AddFile(auditFileBasename, content)
+	}
```

Comment:
You have the `CopyFile` helper to copy a file from the file system to the flare.

```suggestion
	fb.CopyFile(r.auditFilename)
```